spec/factories/budgets.rb

```diff
@@ -71,6 +79,10 @@
     trait :drafting_budget do
       association :budget, factory: [:budget, :drafting]
     end
+
+    trait :unrestricted do
```

Comment:
Ideally we will find a name that can be used as both a trait and an attribute in DB. Maybe `restricted` is the one. 🤔